code/handlers/trackservers.q

```diff
@@ -333,13 +333,24 @@ reqprocsnotconn:{[requiredprocs]
     not all requiredprocs in exec proctype from .servers.SERVERS where .dotz.liveh[w]
   }
 
+startupdependent:{[requiredprocs;timeintv;cycles]
+  if[0<>cycles;startupdepcycles[requiredprocs;timeintv;cycles]];   
+  while[.servers.reqprocsnotconn[requiredprocs];
+     .os.sleep[timeintv];
+     .servers.startup[]]  
+ }
```

Comment:
I don't think this is quite what I meant.  We need to preserve the definition of startupdependent to just try infinitely (as it previously did)

I think startupdependent can just be defined as a projection of startupdepcycles e.g. 

startupdepcycles[;;0W]